modules/oke/nodepools.tf

```
@@ -121,8 +121,8 @@ resource "oci_containerengine_node_pool" "nodepools" {
     ignore_changes = [
       kubernetes_version,
       defined_tags, # automatic tagging after apply
-      node_metadata["user_data"], # templated cloud-init
-      node_config_details["placement_configs"], # dynamic placement configs
+      node_metadata[0].user_data, # templated cloud-init
```

```suggestion
      node_metadata.user_data, # templated cloud-init
```
>│ Error: Unsupported attribute
│ 
│   on modules/oke/nodepools.tf line 124, in resource "oci_containerengine_node_pool" "nodepools":
│  124:       node_metadata[0].user_data, # templated cloud-init
│ 
│ Can't access attributes on a primitive-typed value (string).

Small adjustment for the map field on this one

---

It worked the way I did when I tested it, but it probably depends on how many `node_metadata`s there are.
May as well ignore the whole `node_metadata`.

---

I'm good with ignoring `node_metadata` here.

---

sounds good. I'll merge when completed.